package.json

```diff
@@ -28,7 +28,9 @@
   },
   "dependencies": {
     "dotenv": "^16.0.0",
-    "pg": "^8.7.3"
+    "express": "^4.18.2",
+    "pg": "^8.7.3",
+    "supertest": "^6.3.3"
```

Comment:
this will be a dev dependency, as its not needed for production

Reply:
"express": "^4.18.2",
    "pg": "^8.7.3",
    
these 2 are dependencies